tests/ci/cherry_pick.py

```diff
@@ -67,15 +67,17 @@ class ReleaseBranch:
 ### Note
 
 This pull-request will be merged automatically as it reaches the mergeable state, \
-**do not merge it manually**.
+**do not merge it manually**. It's 100% safe, but completely meaningless.
```

Comment:
For it to be even less scary :)
```suggestion
do not merge it manually. It's 100% safe, but completely meaningless.
```

Reply:
L. 61:
It contains the changes produced by running `git cherry-pick`.